src/main/java/com/SAS/User/FieldRole.java

```diff
@@ -0,0 +1,12 @@
+/**
+ * The class represents the different roles in the field
+ */
+
+package com.SAS.User;
+
+public enum FieldRole {
+    goalKeeper,
```

Comment:
enum convensions in java are capital, please change to GOAL_KEEPER, DEFENDER, STRIKER, MID_FIELDER

Reply:
We'll change it.